packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/model/Product.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:image_cropper/image_cropper.dart';
+
+class ImageUploadCard extends StatefulWidget {
+  const ImageUploadCard({
+    Key key,
+    this.product,
+    this.imageField,
+    this.buttonText,
+  }) : super(key: key);
+
+  final Product product;
+  final ImageField imageField;
+  final String buttonText;
+
+  @override
+  _ImageUploadCardState createState() => _ImageUploadCardState();
+}
+
+class _ImageUploadCardState extends State<ImageUploadCard> {
+  File _image;
+  final ImagePicker picker = ImagePicker();
+
+  Future<void> _getImage() async {
+    print('getImage');
+    final PickedFile pickedFile =
+        await picker.getImage(source: ImageSource.camera);
+    print('got image');
+
+    if (pickedFile != null) {
+      final File croppedImage = await ImageCropper.cropImage(
+        sourcePath: pickedFile.path,
+        androidUiSettings: const AndroidUiSettings(
+          lockAspectRatio: false,
+          hideBottomControls: true,
+        ),
+      );
+      print('cropping image - done');
+
+      if (croppedImage != null) {
+        setState(() {
+          _image = croppedImage;
+        });
+
+        final SendImage image = SendImage(
+          lang: OpenFoodFactsLanguage.ENGLISH,
```

Comment:
Maybe something more like `Localizations.localeOf(context).languageCode`

Reply:
Good point. I changed it to lang: LanguageHelper.fromJson(Localizations.localeOf(context).languageCode)  as openfoodfacts-dart as an Enum for languages instead of just using the 2 letter language code.

I still get a "front_en" image uploaded on the server, even though my languageCode is "fr". Maybe a bug somewhere else.